src/users/migrations/0012_auto_20200712_1632.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.0.3 on 2020-07-12 08:32
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0011_cocrecord_agreed_at'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='user',
+            name='id',
+            field=models.BigAutoField(primary_key=True, serialize=False),
```

Comment:
Do we need `verbose_name='ID'` here?

Reply:
I think it doesn't matter. I suppose `verbose_name` in migrations has no effect.

Reply:
Correct.